### ISSUE TITLE
Could you please provide the pre-trained model file for sRGB?

### ISSUE THREAD

**Issue by xzhub** [491506870/PRIDNet#4]
Looks like PRIDNet is very promising for real world denoising,  I am wondering if you could provide the download link of pre-trained model for sRGB? 

Thanks!

**Comment by 491506870** [comment]
Of course, but may have to wait for some days, the COVID-19 delayed our return to school.

**Comment by bathcenter** [comment]
> 看起来PRIDNet对于现实世界的去噪非常有前途，我想知道您是否可以提供sRGB的预训练模型的下载链接？
> 
> 谢谢！

请问你复现成功了吗？我们可以交流一下吗
